Source/Core/Common/Arm64Emitter.h

```diff
@@ -1137,14 +1138,16 @@ class ARM64CodeBlock : public CodeBlock<ARM64XEmitter>
 private:
   void PoisonMemory() override
   {
-    u32* ptr = (u32*)region;
-    u32* maxptr = (u32*)(region + region_size);
     // If our memory isn't a multiple of u32 then this won't write the last remaining bytes with
     // anything
     // Less than optimal, but there would be nothing we could do but throw a runtime warning anyway.
     // AArch64: 0xD4200000 = BRK 0
-    while (ptr < maxptr)
-      *ptr++ = 0xD4200000;
+    constexpr u32 brk = 0xD4200000;
```

Comment:
Sure, but it doesn't really matter since `::brk(some_ptr);` will still resolve correctly. I'll change it though.